ci/requirements-3.5_OSX.run

```diff
@@ -1,5 +1,5 @@
 pytz
-numpy
+numpy=1.10.4
```

Comment:
git blame tells me this comes from @jreback https://github.com/jrings/pandas/commit/fded94274effa2d592e48ec2079ba3b78e5bb232